src/test/java/seedu/address/model/person/VendorTest.java

```diff
@@ -0,0 +1,109 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.tag.Tag;
+
+/**
+ * Simplified test class for Vendor.
+ * More detailed tests to be written in the future.
+ *
+ * @author Shannon
+ * @version v1.2
+ */
+
+public class VendorTest {
+    @Test
+    public void isSameVendor() {
```

Comment:
If I recall correctly, test names ought to follow the 3 part structure mentioned in the coding standard. Same for the names below.

Reply:
According to the [Java coding standard](https://se-education.org/guides/conventions/java/basic.html), 
> Third part or both second and third parts can be omitted depending on what's covered in the test. For example, the test method `sortList_emptyList()` will test `sortList()` method for all variations of the 'empty list' scenario and the test method `sortList()` will test the `sortList()` method for all scenarios.

Reply:
That's new to me! Thanks for the info Samuel. Do you think this method name can remain the same, or if it should still be changed to reflect the feature under test (i.e. isSamePerson)?

Reply:
That's a good point! Since the feature under test `isSameVendor()` is not implemented in the `Vendor` class, the test method may not be necessary since it is already tested by `Person::isSamePerson`!

Reply:
@keaganpzh your thoughts? Do you think it is necessary to create tests for non-overridden methods?

Reply:
In the spirit of code coverage we should try our best to test as many methods in our classes as possible. CodeCov expects 75% of our codebase to be covered by JUnit tests to reflect a passing status.